src/orange/MatrixUtils.hh

```diff
@@ -91,5 +121,41 @@ CELER_FUNCTION Array<T, N> gemv(T alpha,
     return result;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Naive transposed generalized matrix-vector multiply.
+ *
+ * \f[
+ * z \gets \alpha A^T x + \beta y
+ * \f]
+ *
+ * This should be equivalent to BLAS' GEMV with the 't' option. All
```

Comment:
With the `TRANS` option.

Reply:
The actual BLAS uses a single-character argument to specify this (classic Fortran 🙄 ) though double checking the character should technically be uppercase.